src/overlays/actors/ovl_En_Raf/z_en_raf.h

```diff
@@ -2,16 +2,64 @@
 #define Z_EN_RAF_H
 
 #include "global.h"
+#include "objects/object_raf/object_raf.h"
+
+#define EN_RAF_GET_TYPE(thisx) (((thisx)->params >> 12) & 0xF)
+#define EN_RAF_GET_REVIVE_TIMER(thisx) (((thisx)->params >> 7) & 0x1F)
+#define EN_RAF_GET_SWITCH_FLAG(thisx) ((thisx)->params & 0x7F)
+
+typedef enum {
+    /* 0 */ EN_RAF_TYPE_NORMAL,
+    /* 1 */ EN_RAF_TYPE_ALEADY_DEAD,          // Spawns without trap, so it can't eat bombs/player
```

Comment:
Nice, keeping up the Hokuto no Ken references...

Reply:
I'll be 100% honest, it actually came from Bloodborne: https://www.youtube.com/watch?v=4F54phkQgiU

But that in itself might be a reference to something else. Either way, this is probably getting replaced if I switch from `DeadIdle` to `Dormant`, but it was fun while it lasted.

Reply:
https://knowyourmeme.com/memes/you-are-already-dead-omae-wa-mou-shindeiru